scripts/parser-tests/typescript/allowlist.txt

```diff
@@ -113,6 +114,7 @@ mismatchedClassConstructorVariable.ts
 misspelledNewMetaProperty.ts
 modifiersOnInterfaceIndexSignature1.ts
 moduleDuplicateIdentifiers.ts
+moduleKeywordDeprecated.ts
```

Comment:
Apparently TS reports a type checking error and not a parsing error. I'll make it recoverable for our parser, at least.